docs/ide/index.rst

```diff
@@ -0,0 +1,25 @@
+.. _ide:
+
+IDE Support
+===========
+
+The following supported Sphinx-Needs language features are intergrated into `Esbonio language server <https://github.com/swyddfa/esbonio>`_.
```

Comment:
`The following ... features`? Where are they?
I would think a list is following...